test/firestore_test.dart

```diff
@@ -55,8 +55,7 @@ void main() {
         databaseURL: databaseUrl,
         projectId: projectId,
         storageBucket: storageBucket);
-
-    firestore = fb.firestore();
+    firestore = app.firestore();
```

Comment:
Why this change?

Reply:
Actually I made this change first because in my testing I was initializing another app first and fb.firestore() seems to pick the first one. So I had to explicitly make it point to the correct app and I thought it was more correct to keep it this way. I initialize another app in the same file so I found it clearer to specify the app from which firestore is picked. Do you want me to revert this change?